src/notifications/index.ts

```diff
@@ -157,6 +157,10 @@ export async function generateAndStoreNotificationsV2(
   entityManager: EntityManager,
   args: NotificationHandlerReturn,
 ): Promise<void> {
+  if (!args) {
+    return;
+  }
```

Comment:
This never gets called with `args = undefined` because worker checks for it but for safety its here